private/dev/Dashboard.js

```diff
@@ -47,6 +47,10 @@ const RESTORE = false
 // Rest is implementation! Obviously edit as necessary...
 
 export default () => {
+  const companionAllowedHosts = undefined
+  // If you want to test dynamic OAuth, instead use:
+  // const companionAllowedHosts = Transloadit.COMPANION_PATTERN
```

Comment:
Can we use `import.meta.env.VITE_COMPANION_ALLOWED_HOSTS` instead?

Reply:
Oh, but then we cannot use the special Transloadit.COMPANION_PATTERN constant, so we would need to duplicate that value inside .env, not sure if that's the best solution?

Reply:
`Transloadit.COMPANION_PATTERN` should work for the vbox as well as prod, so I think it's the only one we're likely to use?